pilot/pkg/proxy/envoy/v2/config.go

```diff
@@ -32,34 +30,58 @@ import (
 	google_protobuf "github.com/gogo/protobuf/types"
 	_ "github.com/golang/glog" // nolint
 
+	"time"
+
 	meshconfig "istio.io/api/mesh/v1alpha1"
 	networking "istio.io/api/networking/v1alpha3"
-	routing "istio.io/api/routing/v1alpha1"
+	"istio.io/api/routing/v1alpha1"
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/proxy/envoy/v1"
 	"istio.io/istio/pkg/log"
 )
 
 const (
-	// filterNameRouter is the name for the router filter.
-	filterNameRouter = "router"
-)
+	// RouterFilter is the name for the router filter.
+	RouterFilter = "envoy.router"
 
-// ListListenersResponse returns a list of listeners for the given environment and source node.
-func ListListenersResponse(env model.Environment, node model.Proxy) (*xdsapi.DiscoveryResponse, error) {
-	ls, err := buildListeners(env, node)
-	if err != nil {
-		return nil, err
-	}
+	// HTTPConnectionManagerFilter is the name of HTTP filter.
+	HTTPConnectionManagerFilter = "envoy.http_connection_manager"
 
-	resp := &xdsapi.DiscoveryResponse{}
-	for _, ll := range ls {
-		lr, _ := google_protobuf.MarshalAny(ll)
-		resp.Resources = append(resp.Resources, *lr)
-	}
+	// TCPProxyFilter is the name of the TCP Proxy network filter.
+	TCPProxyFilter = "envoy.tcp_proxy"
 
-	return resp, nil
-}
+	// CORSFilter is the name of the CORS network filter
+	CORSFilter = "envoy.cors"
+
+	// MongoProxyFilter is the name of the Mongo Proxy network filter.
+	MongoProxyFilter = "envoy.mongo_proxy"
```

Comment:
I have all constants defined in `util` in go-control-plane.

Reply:
the current version of go cp we imported does not have these constants. I lost track of this dependency update thing..

Reply:
Hm, ok. I thought I already brought it in. I can follow up later. Less code is better.